supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from conftest import Fields, TestFields
+from django.core.exceptions import FieldDoesNotExist, ValidationError
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        """Test - checks if supplier is saved in DB.
+
+        Args:
+            supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
```

Comment:
How do you know it was not there before the test?

Reply:
good call.
will add a feature in the save supplier to raise an error if object is already there.

Reply:
You dont need to raise an error when a supplier already exist,change the test for something like to check the existing of a supplier before saving it and after saving it(or delete it if it exist before saving it)

Reply:
You dont need to raise an error when a supplier already exist,change the test for something like to check the existing of a supplier before saving it and after saving it(or delete it if it exist before saving it)